packages/grid/x-data-grid/src/components/toolbar/GridToolbarColumnsButton.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import { ButtonProps } from '@mui/material/Button';
+import { useId } from 'react';
```

Comment:
`useId` is only available in React 18, but we still support React 17. We have a helper for that:

```suggestion
import { unstable_useId as useId } from '@mui/material/utils';
```

Reply:
thanks for the suggestion - will fix